modules/ROOT/pages/backup-restore/online-backup.adoc

```diff
@@ -10,6 +10,24 @@ Remember to xref:backup-restore/planning.adoc[plan your backup] carefully and ba
 Note that it is not allowed to take a backup of a database alias, only physical databases can be backed up.
 ====
 
+Before starting a backup, pay attention to your checkpointing policy, as the checkpoint has to be completed before the backup starts.
```

Comment:
I'd phrase it as something like this: 
A backup job will trigger a checkpoint if needed, and won't proceed until the checkpoint is completed. 

Because, even if the database has just finished checkpointing, if there's data, the backup job will trigger a checkpoint again. And depending on how much there is to checkpoint, the time if takes to finish varies.